javascript/example_code/s3/create-presignedURL/ReadMe.md

```diff
@@ -0,0 +1,60 @@
+# README for Generating Pre-signed URLs
+
+## Purpose
+
+This example uses a Node.js function to generate a pre-signed URL that uploads a specified file to a specified S3 bucket. A pre-signed URL grants temporary access to users who don’t have permission to directly run AWS operations in your account. 
+
+## Prerequisites
+
+To build and run this example, you need the following:
+- Node.js. For more information about installing Node.js, see the [Node.js website](https://nodejs.org).
+- The AWS SDK for JavaScript.  For more information about installing the AWS SDK for JavaScript, see see *Step 3* on the [Getting Started in Node.js in the JavaScript SDK Developer Guide](https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/getting-started-nodejs.html).
```

Comment:
(I think there's extra space between "JavaScript." and "For".
also
...JavaScript, see see Step 3 in the Getting Started...